py2-pippkgs_depscipy.spec

```diff
@@ -58,14 +59,18 @@ for pkg in %builddirectpkgreqs ; do
         # https://github.com/jupyter/jupyter_core/issues/55 - now I delete jupyter.py from one of the providers
         #backports is an example directory that can have multiple packages inside
         if [ $f != "backports" ] ; then  
+        if [ $f != "google" ] ; then  
            echo "  Duplicate file found: $f"
            exit 1
         fi
+        fi
 
       fi
     done
     echo "Copying $SOURCE in %{pkgrel}"
-    rsync -av $SOURCE/ %{i}/${PYTHON_LIB_SITE_PACKAGES}/
+#try cp instead of rsync as we don't want to overwrite duplicate directories
+#    rsync -av $SOURCE/ %{i}/${PYTHON_LIB_SITE_PACKAGES}/
+    cp -r ${SOURCE}/* %{i}/${PYTHON_LIB_SITE_PACKAGES}/ 
```

Comment:
@davidlange6 , do we need such a package any more? We can just explicitly add these py2-* packages dependency in cmssw-tool-conf ?